doc/release-notes/rl-2511.section.md

```diff
@@ -66,6 +66,9 @@
   and called `setup.py` from the source tree, which is deprecated.
   The modern alternative is to configure `pyproject = true` with `build-system = [ setuptools ]`.
 
+- `searx` was updated to use `envsubst` instead of `sed` for parsing config environment variables.
+  For information on how to migrate see the [usage](https://github.com/a8m/envsubst) of `envsubst` and view the updated examples.
```

Comment:
```suggestion
```
I don't think that this is a detail people need to know but I like the example idea from rnhmjoj very much and would rather want to have that here instead.

We should probably use the `server.secret_key = "@SEARX_SECRET_KEY@"` setting as an example as I expect most if not all configurations to use that.

Reply:
Did this, please let me know if you have any more suggestions on this. Thanks!